geofront/backends/cloud.py

```diff
@@ -50,18 +50,21 @@
     from typing import MutableMapping, Optional  # noqa: F401
 
 __all__ = ('CloudKeyStore', 'CloudMasterKeyStore', 'CloudMasterPublicKeyStore',
-           'CloudRemoteSet', 'create_compute_driver', 'create_storage_driver')
+           'CloudRemoteSet', 'create_compute_driver', 'create_storage_driver',
+           'create_cloud_master_pubkey_store')
 
 
 def create_compute_driver(provider_name: str, creds: Sequence[str],
                           **driver_kwargs) -> NodeDriver:
+    """A shortcut NodeDriver factory for environment-variable configs"""
```

Comment:
- Please append a period to it.
- `NodeDriver` should be backquoted (<code>:class:\`~libcloud.compute.base.NodeDriver\`</code>).